File: fibonacci-recursive/python/fibonacci-recursive.py
#!/usr/bin/python3

import sys

def get_fib(fib_number):
    if fib_number == 1:
        return 0
    if fib_number == 2:
        return 1
    else:
        return get_fib(fib_number - 2) + get_fib(fib_number - 1)

print(get_fib(int(sys.argv[1])))

File: fibonacci-recursive/php/fibonacci-recursive.php
#!/usr/bin/php
<?php
function getFib($fibNumber)
{
    if($fibNumber == 1 || $fibNumber == 2)
    {
        return 1;
    }
    return getFib($fibNumber - 2) + getFib($fibNumber - 1);
}

$result = getFib($argv[1]);
echo "$result\n";
?>